coderd/coderd.go

```diff
@@ -1378,6 +1379,26 @@ func New(options *Options) *API {
 		r.Get("/swagger/*", swaggerDisabled)
 	}
 
+	additionalCSPHeaders := make(map[httpmw.CSPFetchDirective][]string)
```

Comment:
Let's save a potential allocation or two

```suggestion
	additionalCSPHeaders := make(map[httpmw.CSPFetchDirective][]string, len(api.DeploymentValues.AdditionalCSPPolicy))
```

Reply:
I appreciate your dedication to saving bytes! :smile: